src/Tribe/Attendees.php

```diff
@@ -588,8 +588,35 @@ public function maybe_generate_csv() {
 			// create a file pointer connected to the output stream
 			$output = fopen( 'php://output', 'w' );
 
+			// Get indexes by keys
+			$flip  = array_flip($items[0]);
+			$prime = $flip['Primary Information'];
+			$name  = $flip['Customer Name'];
+			$email = $flip['Customer Email Address'];
+
+			error_log(
```

Comment:
I think this one should be removed. `error_log`

Reply:
Same as other instances of `error_log`

Reply:
dangit...corrected.